integration/networking/bridge_test.go

```diff
@@ -1089,7 +1090,15 @@ func TestProxy4To6(t *testing.T) {
 	inspect := container.Inspect(ctx, t, c, serverId)
 	hostPort := inspect.NetworkSettings.Ports["80/tcp"][0].HostPort
 
-	resp, err := http.Get("http://[::1]:" + hostPort)
-	assert.NilError(t, err)
+	var resp *http.Response
+	addr := "http://[::1]:" + hostPort
+	poll.WaitOn(t, func(t poll.LogT) poll.Result {
+		var err error
+		resp, err = http.Get(addr) // #nosec G107 -- Ignore "Potential HTTP request made with variable url"
+		if err != nil {
+			return poll.Continue("waiting for %s to be accessible: %v", addr, err)
+		}
+		return poll.Success()
+	})
```

Comment:
This part didn't apply clean because `TestProxy4To6` moved to a different file in master, so I just copy/pasta'd the change to the right file in this branch.